# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/api/FileSystemHandle.h
# Commit: e4b3f3d8ba5f
# Full Hash: e4b3f3d8ba5fd951bf79a2014dcee109255c865f
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2022-08-31 21:54:47
# Regressor Bug: 1786465
# File Overlap Count: 1
# Description:
#   Bug 1786465 - Rename POriginPrivateFileSystem to PFileSystemManager; r=dom-storage-reviewers,jesup
#   
#   The child actor will be held by FileSystemManager on the child side and the
#   parent actor already holds FileSystemDataManager, so it makes sense to rename
#   the protocol to PFileSystemManager.
# ==============================================================================

diff -r 3b0da4b4bb83 -r e4b3f3d8ba5f dom/fs/api/FileSystemHandle.h
--- a/dom/fs/api/FileSystemHandle.h	Wed Aug 31 09:34:48 2022 +0000
+++ b/dom/fs/api/FileSystemHandle.h	Wed Aug 31 09:34:48 2022 +0000
@@ -8,7 +8,7 @@
 #define DOM_FS_FILESYSTEMHANDLE_H_
 
 #include "mozilla/dom/FileSystemActorHolder.h"
-#include "mozilla/dom/POriginPrivateFileSystem.h"
+#include "mozilla/dom/PFileSystemManager.h"
 #include "mozilla/Logging.h"
 #include "nsCOMPtr.h"
 #include "nsISupports.h"
@@ -31,7 +31,7 @@
 
 class DOMString;
 enum class FileSystemHandleKind : uint8_t;
-class OriginPrivateFileSystemChild;
+class FileSystemManagerChild;
 class Promise;
 
 namespace fs {
@@ -62,7 +62,7 @@
   already_AddRefed<Promise> IsSameEntry(FileSystemHandle& aOther,
                                         ErrorResult& aError) const;
 
-  OriginPrivateFileSystemChild* Actor() const;
+  FileSystemManagerChild* Actor() const;
 
  protected:
   virtual ~FileSystemHandle() = default;